warehouse/models/mart/transit_database/dim_county_geography.sql

```diff
@@ -0,0 +1,31 @@
+{{ config(materialized='table') }}
+
+WITH int_transit_database__county_geography_dim AS (
+    SELECT *
+    FROM {{ ref('int_transit_database__county_geography_dim') }}
+),
+
+dim_county_geography AS (
+    SELECT
+
+        key,
+        source_record_id,
+        name,
+        fips,
+        msa,
+        caltrans_district,
+        caltrans_district_name,
+        -- rtpa is not a one-to-one relationship
```

Comment:
If this is another array of organization keys, I would remove it from the final dim, and same for MPO... Generally I think we don't want unhandled arrays of keys in these tables